features/product/viewing_products/viewing_product_reviews_on_product_details_page.feature

```diff
@@ -0,0 +1,25 @@
+@viewing_products
+Feature: Viewing product reviews on product's details page
+    In order to know other customer's opinion about product
+    As a Customer
+    I want to read product reviews on product's details page
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has customer "Howard Phillips Lovecraft" with email "h.p.lovecraft@arkham.com"
+        And the store has customer "Robert E. Howard" with email "robert.e.howard@conan.com"
+        And the store has customer "J.R.R Tolkien" with email "jrr.tolkien@middle-earth.com"
+        And the store has customer "Terry Pratchett" with email "sir.terry@pratchett.com"
```

Comment:
Do we need to create customers? IIRC we have a transformer that creates a customer if needed.